yo/services/blockchain_follower.py

```diff
@@ -51,18 +51,18 @@ def __init__(self, **kwargs):
         self.db.create_notification(**data)
 
     async def handle_vote(self, op):
-        logger.debug('handle_vote received %s op', ['op'][0])
+        logger.info('handle_vote received %s op', ['op'][0])
         vote_info = op['op'][1]
-        logger.debug('Vote on %s (written by %s) by %s with weight %s',
-                     vote_info['permlink'], vote_info['author'],
-                     vote_info['voter'], vote_info['weight'])
+        logger.info('Vote on %s (written by %s) by %s with weight %s',
+                    vote_info['permlink'], vote_info['author'],
+                    vote_info['voter'], vote_info['weight'])
         await self.store_notification(
             trx_id=op['trx_id'],
             from_username=vote_info['voter'],
             to_username=vote_info['author'],
             json_data=json.dumps(vote_info),
             notify_type=VOTE,
-            priority_level=Priority.LOW)
+            priority_level=int(Priority.LOW.value))
```

Comment:
If `Priority` is defined as `enum.IntFlag` class, then there is not need to make this change, or any of the similar changes below.